master/components/dashboard/js/modules/controllers/cAdvisorController.js

```diff
@@ -22,7 +22,11 @@ app.controller('cAdvisorController', [
         $scope.minions = res;
 
         var promises = lodash.map(res.items, function(m) {
-            return cAdvisorService.getDataForMinion(m.metadata.name, (m.status.conditions[0].status === 'True'));
+            var readyCondition = m.status.conditions.filter(function ( obj ) {
+                return obj.type === 'Ready';
+            })[0];
```

Comment:
`...[0]` will give undefined if there is no such condition. Just check for that case below: `readyCondition && readyCondition.status === 'True'`.